i18n/locales/en.ts

```diff
@@ -1729,7 +1735,21 @@ const translations = {
       paragraph_one:
         'You escape the compound and return home. Investors sell their BitRey shares en masse when the story drops, plummeting Vanderpoole’s net worth.',
     },
-    resources: {},
+    resources: {
+      mempool_transaction_one: {
+        bytes_v_weight_heading: 'Bytes vs Weighted Units',
+        bytes_v_weight_paragraph_one:
+          'In Bitcoin block construction, "weighted units" (WU) and "bytes" are two different metrics used to measure the size of transactions and blocks.',
+        bytes_v_weight_subheading_bytes: 'Bytes',
+        bytes_v_weight_paragraph_two:
+          'Bytes refer to the raw size of a transaction or block in terms of actual data. It is the literal size of the transaction data when it is serialized (converted into a format that can be stored or transmitted). Before the implementation of Segregated Witness (SegWit), the block size was limited to 1 megabyte (1 MB), measured in bytes.',
+        bytes_v_weight_subheading_wu: 'Weighted Units',
+        bytes_v_weight_paragraph_three:
+          'Weighted units are a more complex metric introduced with SegWit in Bitcoin Improvement Proposal 141 (BIP 141). This system aims to more fairly allocate block space by considering the impact of witness data separately.',
```

Comment:
recommend link to https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#user-content-Other_consensus_critical_limits

Reply:
Might also mention the new segwit limit is 4000000 weight units since you mentioned the legacy 1 MB limit in the last paragraph.